src/MoBi.Core/Domain/Model/MoBiSimulation.cs

```diff
@@ -1,5 +1,6 @@
 using System.Collections.Generic;
 using System.Linq;
+using FluentNHibernate.Conventions;
```

Comment:
Please remove. This has nothing to do in s simulation file

Reply:
removed